### PR TITLE
Create database

### PR DESCRIPTION
**Pull request by dansaa8**
Database working as intended via cloud with MongoDB ATLAS.

DB collections: Bookings, Users & Movies.

Temporary routes have been made for each Schema/Collection as to initiate them in the ATLAS database, but also for testing to see if they work as intended.

The routes are as follows:
api/collectionName/GET,
api/collectionName/POST

The routes described above are to be deleted later on as the "real" routes described in doc.md(located in rootDir) are implemented continually. But meanwhile they serve as great templates when the "real" routes are created but also to show that the communication with the collections are working properly.

During the codereview, you can use the .JSON-templates in directory: 
'models/post_template.json' to create a new Movie/User/Booking with Postman or whatever you prefer. Feel free to change the properties when you are posting so we don't get duplicates. ;)

### CODE REVIEW

**Review by a-jaxell — COMMENTED** [review]
Nice! Jag tänker bara att `screenings` kanske borde vara mer utförlig än bara en array med datum?

Men å andra sidan så är ju tidpunkten en film visas en unik indikator i sig så länge man inte ska ha mer än en visning för en film på samma klockslag.

File: src/pages/api/bookings/GET.ts
Comment (KRQQN) [inline src/pages/api/bookings/GET.ts:1]: hämtar denna alla bokningar för alla filmer? :P
hur ska denna användas?

File: src/pages/api/users/POST.ts
Comment (KRQQN) [inline src/pages/api/users/POST.ts:1]: Denna kommer vara överflödig, login/create user har jag redan börjat och är snart klar med!

**Review by KRQQN — APPROVED** [review]

File: src/pages/api/users/POST.ts
Comment (dansaa8) [reply src/pages/api/users/POST.ts:1]: Om du läser mitt meddelande i pull-requesten så kommer du förstå. Jag nämner varför jag har lagt till dem det 2 ggr i texten. ^^

File: src/pages/api/bookings/GET.ts
Comment (dansaa8) [reply src/pages/api/bookings/GET.ts:1]: Läs meddelandet jag skrev i pull-requesten. :P

**Pull request closed without merging**